<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZOpenCVStereoMultiCameraCalibratorConfigWidget</class>
 <widget class="QWidget" name="ZOpenCVStereoMultiCameraCalibratorConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Other options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Maximum iterations</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="termMaxIterationsSpinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Termination epsilon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="termEpsilonSpinBox">
        <property name="decimals">
         <number>15</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
        <property name="value">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="distortionModelGroupBox">
     <property name="title">
      <string>Distortion model</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="zeroTangentialDistortionCheckBox">
        <property name="text">
         <string>Zero tangential distortion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="useRationalModelCheckBox">
        <property name="text">
         <string>Use rational model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="fixK1CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="fixK2CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="fixK3CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="fixK4CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K4</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="fixK5CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K5</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="fixK6CheckBox">
        <property name="text">
         <string>Fix radial distortion coefficient K6</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Stereo calibration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="fixIntrinsicCheckBox">
        <property name="toolTip">
         <string>Use this if  you only want to calculate the rotation and translation between cameras</string>
        </property>
        <property name="text">
         <string>Keep current camera calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="cameraMatrixGroupBox">
     <property name="title">
      <string>Camera matrix</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="fixPrincipalPointCheckBox">
        <property name="text">
         <string>Fix principal point</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="fixAspectRatioCheckBox">
        <property name="text">
         <string>Fix aspect ratio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="sameFocalLengthCheckBox">
        <property name="text">
         <string>The cameras have the same focal length</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
